pkg/controller/controlplane/valuesprovider.go

```diff
@@ -370,6 +370,16 @@ func (vp *valuesProvider) GetControlPlaneShootChartValues(
 	return getControlPlaneShootChartValues(cluster)
 }
 
+// GetControlPlaneShootCRDsChartValues returns the values for the control plane shoot CRDs chart applied by the generic actuator.
+// Currently the provider extension does not specify a control plane shoot CRDs chart. That's why we simply return empty values.
+func (vp *valuesProvider) GetControlPlaneShootCRDsChartValues(
+	_ context.Context,
+	_ *extensionsv1alpha1.ControlPlane,
+	_ *extensionscontroller.Cluster,
+) (map[string]interface{}, error) {
+	return map[string]interface{}{}, nil
+}
+
```

Comment:
Hm, probably this should be added to https://github.com/gardener/gardener/blob/master/extensions/pkg/controller/controlplane/genericactuator/noopvaluesprovider.go, or?

Reply:
Nice catch, thank you! It seems that I missed this in the initial PR in the extension library. I now opened https://github.com/gardener/gardener/pull/3755.